portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.servlets;
```

Comment:
Make sure not to include new feature in to existing pull request's that is under review next time.

Reply:
Will do. Sorry for the confusion